store/tikv/region_request.go

```diff
@@ -973,26 +973,24 @@ func (s *RegionRequestSender) onRegionError(bo *Backoffer, ctx *RPCContext, req
 		logutil.BgLogger().Warn("tikv reports `RaftEntryTooLarge`", zap.Stringer("ctx", ctx))
 		return false, errors.New(regionErr.String())
 	}
-	// A stale read request may be sent to a peer which the data is not ready yet, we should retry in this case.
+	// A request may be sent to a peer which the data is not ready yet because the region is merging or splitting,
```

Comment:
Can region split/merge affect it? I changed the error handler here https://github.com/pingcap/tidb/pull/25165/files#diff-5a43205a0a94d800df8c5e543c564d0acb8662a57431b25140b2ecb764b4cb7eR1058.